docs/pages/dev/reference/API/org_unit_registry.md

```diff
@@ -1,50 +1,44 @@
-# Prerequisite
+# `OrgUnitChangeRequest` API
 
-## Possible fields
-- `parent_id`
-- `name`
-- `opening_status`
-- `org_unit_type_id`
-- `groups`
-- `geo_point`
-- `shape`
-- `instances`
+"Change Requests" can be submitted for an `OrgUnit` and then reviewed (approved or rejected).
 
-# POST /api/orgunits/changes/?app_id=..
+This API allows the DHIS2 Health pyramid to be updated using Iaso.
 
-API used to create or modify an OrgUnit with validation (see feature flags)
-When creating an OrgUnit, the client doesn't know the ID or the OrgUnit was not created yet and we might want to apply 
-some changes still. Therefore, the IDs can be either the DB id or the UUID generated by the client.
-Changes that are not provided are considered not being part of the change request.
+The Django model that stores "Change Requests" is `OrgUnitChangeRequest`.
+
+# Create or update an `OrgUnitChangeRequest` object - Web + Mobile (IA-2421)
+
+- `POST /api/orgunits/changes/?app_id=…`
+- `PUT /api/orgunits/changes/?app_id=…`
 
 ## Permissions
 
-User must be authenticated.
+- User must be authenticated
+- User must have the `ORG_UNITS_CHANGE_REQUEST` permission
 
 ## Query parameters
 
-- app_id: String - Optional, project for which this is created.
+- `app_id`: String - Optional, project for which this is created.
 
 ## Body
 
 ```json
 {
   "uuid": "UUID - Client generated UUID",
   "org_unit_id": "String - id or UUID of the OrgUnit to change",
-  "created_at": "Timestamp - format same as /api/instances",
-  "updated_at": "Timestamp - format same as /api/instances",
   "new_parent_id": "String? - id or UUID of the parent OrgUnit, may be null or omitted.",
   "new_name": "String? - Name of the OrgUnit, may be null or omitted.",
   "new_org_unit_type_id": "Int? - id of the OrgUnitType, may be null or omitted",
   "new_groups": "Array of Group ids? - can be empty, null or omitted. Empty means we want to remove all values",
-  "new_geo_point": { "": "New geopoint for the OrgUnit, may be null or omitted",
+  "new_location": { "": "New geopoint for the OrgUnit, may be null or omitted",
     "latitude": "Double - New latitude of the OrgUnit",
     "longitude": "Double - New longitude of the OrgUnit",
-    "altitude": "Double - New altitude of the OrgUnit",
-    "accuracy": "Double - New accuracy of the OrgUnit",
+    "altitude": "Double - New altitude of the OrgUnit"
   },
-  "new_reference_instances":  "Array of instance ids? - may be null or omitted, cannot be empty"
-  }
+  "new_location_accuracy": "Double - New accuracy of the OrgUnit",
+  "new_opening_date": "Timestamp",
+  "new_closed_date": "Timestamp",
```

Comment:
```suggestion
      "new_location_accuracy": "Double? - New accuracy of the OrgUnit, cannot be null nor omitted if `new_location` is provided" ,
      "new_opening_date": "Timestamp? - format same as /api/instances",
      "new_closed_date": "Timestamp? - format same as /api/instances",
```

Reply:
`new_location_accuracy` being null or omitted while `new_location` is provided, should be added in the reasons for a 400

Reply:
Since this endpoint is shared, this might not work for the Web, because `new_location_accuracy` is only provided by the mobile.